release/BUILD

```diff
@@ -58,42 +58,49 @@ py_test(
 
 py_test(
     name = "test_alerts",
-    tags = ["team:ml", "release_unit"],
+    tags = ["team:ci", "release_unit"],
```

Comment:
I know there's no "CI team", but team:ml seemed wrong, too, so let's use that for now.